.github/workflows/check_unit_tests.yml

```diff
@@ -13,50 +12,35 @@ jobs:
 
     services:
       postgres:
-        image: postgres:13  # Use the latest version of PostgreSQL you need
+        image: postgres:13
         ports:
           - 5432:5432
         env:
-          POSTGRES_USER: postgres
           POSTGRES_PASSWORD: pw
-          POSTGRES_DB: testdb
-        options: >-
```

Comment:
For my own learning, what tests were these that were removed and why?

Reply:
It seemed redundant.  We shouldn't have to tell the service to wait AND tell the tests to wait (lines 36-38) for postgres to be available.  Having the latter seemed sufficient.